ethpm/package.py

```diff
@@ -73,3 +79,20 @@ def name(self):
     @property
     def version(self):
         return self.package_data['version']
+
+    #
+    # Deployments
+    #
+
+    def get_deployments(self, w3):
+        """
+        API to retrieve instance of deployed contract dependency.
+        """
+        validate_w3_instance(w3)
+        validate_deployments_are_present(self.package_data)
```

Comment:
Checks that `"deployments"` key is present and that it's not empty. Is that too much / breaking the `1 thing / line of code` rule?

Reply:
nah, that's fine.  functions are *wrappers* around chunks of functionality so they get a pass on that rule/guideline.